socketio/src/lib.rs

```diff
@@ -124,4 +124,19 @@ pub(crate) mod test {
 
         url
     }
+
+    // The socket.io auth server for testing runs on port 4204
+    const AUTH_SERVER_URL: &str = "http://localhost:4204";
+
+    pub(crate) fn socket_io_auth_server() -> Url {
+        let url =
+            std::env::var("SOCKET_IO_AUTH_SERVER").unwrap_or_else(|_| AUTH_SERVER_URL.to_owned());
```

Comment:
Could you please also add the `SOCKET_IO_AUTH_SERVER` env variable here: https://github.com/1c3t3a/rust-socketio/blob/3dce3e1e690847676a2e465adc49d0483e181aed/.devcontainer/docker-compose.yaml#L45